cpp/benchmarks/tpch/q1.cpp

```diff
@@ -108,8 +107,6 @@ int main(int argc, char const** argv)
   auto resource = create_memory_resource(args.memory_resource_type);
   rmm::mr::set_current_device_resource(resource.get());
 
-  cudf::examples::timer timer;
```

Comment:
Timers have been removed for now, because a following PR would convert these into NVBench's.